index.ts

```diff
@@ -9,14 +9,32 @@ const bucketName: string = process.env.BUCKET_NAME as string;
 const getImage = async (event: EventDto): Promise<any> => {
 	try {
 		// parse params
+		const imageKey: string | undefined = event?.pathParameters?.imageName;
+		if (!imageKey) return {statusCode: 400, body: 'Image name is required'};
+
+		const [_, imageExtension] = imageKey.split('.');
+
+		if (imageExtension !== 'jpg' && imageExtension !== 'jpeg' && imageExtension !== 'png')
+			return {statusCode: 400, body: 'Image extension not supported'};
+
+		const queryStringParameters: { width: string, height: string, v: string } = event?.queryStringParameters;
+		const width: number = parseInt(queryStringParameters?.width);
+		const height: number = parseInt(queryStringParameters?.height);
```

Comment:
should use convert function instead of parse function
e.g Number(queryStringParameters.width)